sonar-server/time-travel-server.go

```diff
@@ -82,12 +84,56 @@ func (s *timeTravelServer) Start() {
 	log.Println("start timeTravelServer...")
 }
 
+func toLowerMap(m map[string]interface{}) {
+	for key, val := range m {
+		switch v := val.(type) {
+		case map[string]interface{}:
+			if e, ok := m[key].(map[string]interface{}); ok {
+				toLowerMap(e)
+				m[strings.ToLower(key)] = e
+				if strings.ToLower(key) != key {
+					delete(m, key)
+				}
+			} else {
+				log.Println("bad")
+			}
+
+		case []interface{}:
+			if e, ok := m[key].([]interface{}); ok {
+				for idx := range e {
+					switch e[idx].(type) {
+					case map[string]interface{}:
```

Comment:
You might also need to consider the case where you have list in list `case []interface{}:`.
If you do not want to handle list in list now, just leave an error message and exit in that case